test/integration/test_integration.py

```diff
@@ -163,7 +196,7 @@ def test_run_override(mock_datetime, with_concept_maps):
 expected_diagnostic_report_count = pd.read_sql_query(
     "SELECT COUNT(*) FROM diagnoses_icd", con=engine
 ).at[0, "count"]
-expected_practitioner_count = pd.read_sql_query("SELECT COUNT(*) FROM caregivers", con=engine).at[
+expected_practitioner_count = pd.read_sql_query("SELECT COUNT(*) FROM admissions", con=engine).at[
```

Comment:
that would be `expected_admissions_count` then ?

Reply:
I did a mapping that doesn't mean anything where generalPractitioner references point to the table admissions...
But I'll change to expected_admissions_count anyway